Adding new concepts and procedures for Multi CV hosts and AKs (#3609)
theforeman/foreman-documentation: guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

> [id="Viewing_multiple_content_view_environment_hosts{context}"]

Reviewer: ```suggestion
[id="viewing-multiple-content-view-environment-hosts"]
```

= Viewing multiple content view environment hosts

> Use this procedure to view hosts with multiple content view environments or multi-environment hosts in the {ProjectWebUI} and Hammer CLI.

Reviewer 2: ```suggestion
Use this procedure to view hosts with multiple content view environments, or multi-environment hosts, in the {ProjectWebUI} and Hammer CLI.
```

.Prerequisites
Ensure that you have at least one multi-environment host. 

.Procedure
. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
+
[NOTE]
====
If you have the *Show new host overview page* setting enabled, the new UI appears by default.
====
+

> . Click *Manage columns*.

Reviewer 2: If you _don't_ have that setting enabled, you still have to click the New UI button.

. Under *Content*, click the *Content view environments* checkbox and click *Save*.
. Content view environments for each host are displayed under the *Content view environments* column.
. Click on the content view environments to display details.

On the *Host Details* page, the *Content view environments* card displays the details of the content view environments.

[NOTE]
====

> If you reassign the content view environment of a multi-environment host from the *Content view environments* card, the host's content view environments are overwritten and is no longer a multi-environment host. 

Reviewer 2: ```suggestion
If you reassign the content view environments of a multi-environment host from the *Content view environments* card, the host's content view environments are overwritten and it is no longer a multi-environment host. 
```

If you reassign the content view environments of a multi-environment host from the *Content view environments* card, the host's content view environments are overwritten and it is no longer a multi-environment host. 
====

.CLI procedure

> . Run the following command to view a list of hosts in your organization:

Reviewer: ```suggestion
. View a list of hosts in your organization:
```

suggestion: reword

Avoid using "the following command" because it does not add anything useful to the docs. The sentence ends with a colon and code block which makes it obvious that users have to run this. The sentence also uses imperative style AFAICT.

+
[options="nowrap" subs="+quotes"]
----

> $ hammer host list --organization “Default Organization”

Reviewer 2: italicize example values

> $ hammer host list --organization _“Default Organization”_

Reviewer: ```suggestion
$ hammer host list --organization "_My_Organization_"
```

suggestion: use normal double quotes and a generic label.

----

> The `CONTENT VIEW ENVIRONMENTS` column displays the labels of content view environments associated with each host. The `MULTI CONTENT VIEW ENVIRONMENT` column shows `Yes` for multi-environment hosts.

Reviewer: ```suggestion
The `CONTENT VIEW ENVIRONMENTS` column displays the labels of content view environments associated with each host.
The `MULTI CONTENT VIEW ENVIRONMENT` column shows `Yes` for multi-environment hosts.
```

suggestion: use one sentence per line.

+
. Run the following command to view multi-environment information for a host:
+
[options="nowrap" subs="+quotes"]
----

> $ hammer host info --id

Reviewer 2: missing example ID value (remember, IDs are numbers)

> $ hammer host info --id _'My_Host_ID'_

Reviewer: ```suggestion
$ hammer host info --id _My_Host_ID_
```

> $ hammer host info --id _[1,2,3]_

Reviewer 2: ```suggestion
$ hammer host info --id _'My_Host_ID'_
```

----

> The information for the host is displayed including multiple content view environments.

Reviewer: ```suggestion
+
The information for the host is displayed including multiple content view environments.
```

suggestion: show this line on the same level of indentation as the code block.

[id="Viewing_multiple_content_view_environment_hosts{context}"]

> = Viewing multiple content view environment hosts

Reviewer 2: ```suggestion
= Viewing multi-environment hosts
```

Author: I'm going to keep it spelled out in the titles and then mention it in the first instance.

> Use this procedure to view multiple content view environment activation keys in Hammer CLI

Reviewer 2: ```suggestion
Use this procedure to view multi-environment hosts in Hammer CLI.
```

Author: Changing it to `Use this procedure to view activation keys with multiple content view environments or multi-content activation in the {ProjectWebUI} and Hammer CLI.`

Author: Changing it to `Use this procedure to view hosts with multiple content view environments or multi-content hosts in the {ProjectWebUI} and Hammer CLI.`  Same with activation keys.

.Prerequisites

> Register a host with the created activation key. 

Reviewer 2: This is not the only way to end up with a multi-environment host. You can also (re)assign a host to multiple content view environments after registration, as described elsewhere in this PR.

```suggestion
Have at least one multi-environment host. 
```

> For more information, see xref:Using_Activation_Keys_for_Host_Registration_{context}[Using activation keys for host registration].

Reviewer 2: ```suggestion
```

> [NOTE]
> ====
> Click the *Insecure* checkbox when registering a host.
> ====

Reviewer 2: This is part of global registration, which is also documented elsewhere and doesn't need to be mentioned here.
```suggestion
```

.Procedure
. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.

> . Click the *New UI* button.

Reviewer 2: If you have the 'Show new host overview page' setting on, you will see the new UI by default. Not sure how this should be handled in the docs.

Author: I added it as a note after the first step to say `If you have the *Show new host overview page* setting enabled, the new UI appears by default.`

. Click *Manage columns*.
. Under *Content*, click the *Content view environments* checkbox and click *Save*.

> . The multiple content view environments for the host are displayed under the *Content view environments* column.

Reviewer 2: The column displays information for both single-environment and multi-environment hosts.

```suggestion
. Content view environments for each host are displayed under the *Content view environments* column.
```

. Click on the content view environments to display details.

> . Click on the vertical ellipsis to the right of the content view environments and select *Change content view environments” from the drop-down menu.

Reviewer 2: ```suggestion
```

This procedure is about viewing, not changing.

> [WARNING]
> ====
> This host is associated with multiple content view environments. 
> If you assign a lifecycle environment and content view here, the host will be removed from the other environments.
> ====

Reviewer 2: ```suggestion
```
The procedure is about viewing, not changing.

> On the *Hosts Details* page, the *Content view environments* card displays the details of the content view environments. 

Reviewer 2: ```suggestion
On the *Host Details* page, the *Content view environments* card displays the details of the content view environments. 
```

Reviewer 2: If you want to include a note here, I wouldn't be opposed.
```suggestion
On the *Hosts Details* page, the *Content view environments* card displays the details of the content view environments. 

[WARNING]
====
If you reassign the content view environment of a multi-environment host from this screen, the host's content view environments will be overwritten and it will no longer be a multi-environment host. Instead, reassign multi-environment hosts' content view environments using Hammer.
====
```

.CLI procedure

> . Run the following command to view the host:

Reviewer 2: ```suggestion
. Run the following command to view a list of hosts in your organization:
```

+
[options="nowrap" subs="+quotes"]
----
$ hammer host list --organization “Default Organization”
----
This lists multiple content view environments for the host.
+
. Run the following command to view host information:
+
[options="nowrap" subs="+quotes"]
----
$ hammer host info –_”Information_ID”_
----

> The information for the host is displayed including multiple content view environments.

Reviewer 2: Same comments as about the activation key procedure above:
- call out the columns
- fix –_”Information_ID”_
